app/views/shared/_breadcrumbs.html.erb

```diff
@@ -17,7 +17,7 @@
         <li class="breadcrumbs__item" itemscope itemtype="http://data-vocabulary.org/Breadcrumb">
           <span class="visually-hidden">&gt;</span>
             <span itemprop="title">
-              <%= t(last_link[:locale_key]) %>
+              <%= (last_link[:locale_key] && t(last_link[:locale_key])) || last_link[:name] %>
```

Comment:
This is a little hard to parse — could we make it clearer? maybe through a helper if nothing else makes sense